sklearn/preprocessing/data.py

```diff
@@ -480,6 +480,14 @@ def minmax_scale(X, feature_range=(0, 1), axis=0, copy=True):
 class StandardScaler(BaseEstimator, TransformerMixin):
     """Standardize features by removing the mean and scaling to unit variance
 
+    The standard score of a data point `x` is calculated as:
+
+        z = (x - u) / std
+
+    where `u` is the mean of the population or zero if `with_mean=False`, and
+    `std` is the standard deviation of the population or one if
```

Comment:
I would either use explicit names, `mean` and `std`, or single letters as in math expressions, `u` and `s`, but not a mix.

Reply:
Good call, @TomDLT. I was initially considering using `μ` and `σ`, but I'm not sure how they'd be rendered. I'm sticking to 'u' and 's'.